src/components/CounterIcon/CounterIcon.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 
-const CounterIcon = (props) => {
+const CounterIcon = ({counterIcon}) => {
```

Comment:
Personally, I'm a fan of this shortcut notation, if you know exactly what you want to pull off the incoming parameter object. I'd just suggest adding a comment indicating it's expecting the incoming parameter to be `props`, this way if I'm looking at the function, and not where it's being called, I can better tell what it's intended to do.

Reply:
I completely agree, my number one weakness is documentation and commenting.